stdlibs/stdlibs.go

```diff
@@ -141,6 +141,61 @@ func InjectPackage(store gno.Store, pn *gno.PackageNode) {
 				m.PushValue(res0)
 			},
 		)
+		// TODO: IBC handshake process
+		// this is called mannuly, maybe should have some modifier for the contract,like `IBC`,
+		// so the IBC channel is initialized during deploying, and can be used more efficiently
```

Comment:
The IBC channel should be chain-managed, with a consistent queue creation method initiated by the contract or the chain.